### PR TITLE
STR-239 -- FE: Go through 2nd revisions for all modules

### PR DESCRIPTION
**Pull request by BrianLinggadjaja**
# Description
Revised all modules with the 2nd revisions requested.
  
# Testing
1. Ensure all the content revision are made, **I stroked through them on the box.**

### CODE REVIEW

File: resources/js/components/modules/data/modules/alcoholModule.js [hunk 1]
```diff
@@ -1248,12 +1249,12 @@ export const alcoholModuleSlides = {
...
     slide_type: 'emailForm',
     header: {
-      header: 'Set a wellness goal!',
+      header: 'Set a Wellness Goal!',
       sub_header: 'Now is a great time to set a wellness goal to practice responsible drinking; or if you don\'t drink, then to improve your overall health!',
       tooltip: null
     },
     content: {
-      textAreaPlaceHolder: 'For example, "Next week I will plan to drink one fewer drink than normal when I go out with friends." Or, \n "I\'ll work out at least once next week for 30 minutes!"'
+      textAreaPlaceHolder: 'i.e. "Next week I will plan to drink one fewer drink than normal when I go out with friends." Or, \n "I\'ll work out at least once next week for 30 minutes!"'
```
Comment (JaimeGG64) [inline resources/js/components/modules/data/modules/alcoholModule.js]: Can you remove the `\n`.

**Review by JaimeGG64 — DISMISSED** [review]
Can you remove the `/n` on the `slide_type: 'emailForm'` for the Alcohol, Marijuana, Nicotine Module

File: resources/js/components/modules/data/modules/marijuanaModule.js [hunk 2]
```diff
@@ -1064,11 +1064,11 @@ export const marijuanaModuleSlides = {
     slide_type: 'emailForm',
     header: {
       header: 'Set a wellness goal!',
-      sub_header: 'Now is a great time to set a wellness goal to naturally increase your dopamine!​ &nbsp;\n Or,\n If you use marijuana, what are some steps that you can take to reduce use, or use more responsibly?',
+      sub_header: 'Now is a great time to set a wellness goal to naturally increase your dopamine!​&nbsp;Or, If you use marijuana, what are some steps that you can take to reduce use, or use more responsibly?',
       tooltip: null
     },
     content: {
-      textAreaPlaceHolder: 'For example, "Next week I will listen to classical music for 10 minutes a day to increase my dopamine levels." Or, \n "Next week I will politely decline using marijuana if a friend asks by saying \'No thanks, I\'m driving\'."'
+      textAreaPlaceHolder: 'i.e. "Next week I will listen to classical music for 10 minutes a day to increase my dopamine levels." Or, \n "Next week I will politely decline using marijuana if a friend asks by saying \'No thanks, I\'m driving\'."'
```
Comment (JaimeGG64) [inline resources/js/components/modules/data/modules/marijuanaModule.js]: Marijuana has a `\n`

Comment (BrianLinggadjaja) [reply resources/js/components/modules/data/modules/marijuanaModule.js]: I think I got them all now

**Review by JaimeGG64 — APPROVED** [review]
💵

**Pull request merged**